src/SFA.DAS.Roatp.CourseManagement.Web/Models/AddAStandard/ConfirmNonRegulatedStandardViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SFA.DAS.Roatp.CourseManagement.Web.Models.Standards;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.Models.AddAStandard
+{
+    public class ConfirmNonRegulatedStandardViewModel : ConfirmNonRegulatedStandardSubmitModel
+    {
+        public StandardInformationViewModel StandardInformation { get; set; }
+        public string CancelLink { get; set; }
+    }
+
+    public class ConfirmNonRegulatedStandardSubmitModel
```

Comment:
just a suggestion: put into own file

Reply:
I think it is ok to be in there. I have all the submit models in the same file so it is easy to work with view model, otherwise you are editing two files.